script.js

```diff
@@ -144,4 +144,33 @@ document.querySelectorAll('.work-btn').forEach((btn) => {
 
 close.addEventListener('click', () => {
   workDetail.style.display = 'none';
+});
+
+// validate email
+const form = document.querySelector('#form-section form');
+const { email } = form.elements;
+
+function showMsg(input, msg) {
+  const small = document.createElement('small');
+  small.classList.add('error');
+  small.textContent = msg;
+  form.insertBefore(small, input);
+}
+
+function resetMsg() {
+  document.querySelectorAll('small').forEach((s) => s.remove());
+}
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  resetMsg();
+
+  if (email.value !== email.value.toLowerCase()) {
+    showMsg(email, 'Your email is not valid, it should be lowercase');
+    return false;
+  }
```

Comment:
Hello @HENRYKC24, thank you for your review and for your time. 
It is working on my own and here is the test screenshot, I can't submit it. Would you please retest... it is the same version and the same branch !!!  
<img width="1169" alt="Screenshot 2021-12-16 at 16 40 02" src="https://user-images.githubusercontent.com/17196048/146403549-aa660623-7a46-40a5-b87d-ede13ce74e38.png">

<img width="1201" alt="Screenshot 2021-12-16 at 16 42 24" src="https://user-images.githubusercontent.com/17196048/146403596-2ac4134d-71f4-4b50-8466-e12a4e45464c.png">

Reply:
Sorry for replying late. I will do that really quick. Just a second...

Reply:
I think you have used the wrong branch!!

Reply:
I just realized your live link in the readme file was not up to date. That was what caused the issue. Sorry about that. Just submit another pull request and I believe it will be approved immediately.
Keep up the good work, @selma-belhadj :muscle: